src/lib/gridsterSelection.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable, EventEmitter } from '@angular/core';
+import { GridsterItem, GridsterItemComponentInterface } from 'lib';
+
+@Injectable()
+export class GridsterSelectionService {
+    public selectItem = new EventEmitter<GridsterItemComponentInterface>();
```

Comment:
don't really like EventEmitters last time I played with them where creating memory leaks.
I prefer callbacks

Reply:
Agree with you, depends on the case how you use them. I will add an unsubscribe in the ngOnDestroy function.